src/pages/ja/bubblegum-v2/hashed-nft-data.md

```
@@ -0,0 +1,49 @@
+{% node label="名前" /%}
+{% node label="シンボル" /%}
+{% node label="URI" /%}
+{% node label="販売者手数料ベーシスポイント" /%}
+{% node label="主要販売が発生" /%}
+{% node label="可変性" /%}
+{% node label="トークン標準" /%}
+{% node label="コレクション" /%}
+{% node label="作成者" /%}
+{% /node %}
+
+{% node #seller-fee-basis-points parent="metadata" y="305" label="販売者手数料ベーシスポイント" theme="blue" /%}
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)